.github/ISSUE_TEMPLATE/07-feature_request.yml

```diff
@@ -0,0 +1,33 @@
+name: Feature Request
+description: requests for new ONNX Runtime features
+assignees: "sophies927"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before submitting your request, please review past submissions to ensure that it is not a duplicate of a known feature request.
+  - type: textarea
+    id: fr-descrip
+    attributes:
+      label: Describe the feature request
+      placeholder: A clear and concise description of the feature you would like and why it should be added.
+    validations:
+      required: true
+  - type: input
+    id: ort-version
+    attributes:
+      label: ONNX Runtime Version
+      placeholder: ex. 1.12.1
+    validations:
+      required: true
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: Alternatives

```

Comment:
Add this text into the pre-filled text box of the description instead of a standalone field